lib/resin-lint.ts

```diff
@@ -210,6 +230,8 @@ export const lint = function(passedParams: any) {
 
 			const paths = options.argv._;
 
+			resinLintConfiguration.prettierCheck = !options.argv['no-prettier'];
```

Comment:
You set `prettierCheck` defaults in `configurations` above. With this line, won't those always be overridden? That makes the defaults unnecessary, and also means the CS default is actually true, not false as in the configuration.